### PR TITLE
fix: word wrap in SendActivity

### LINKED ISSUE #2875: Improve SendActivity word wrap
## Is your feature request related to a problem? Please describe.

SendActivity body text doesn't wrap intelligently

## Describe the solution you'd like

I'd like SendActivity text to break to a new line instead of splitting a word

## Additional context

See attached
Screen Shot 2020-05-03 at 8 20 17 PM

### LINKED ISSUE #2907: Cannot use multi-selection when Visual Editor container is focused
## Describe the bug
Observed by Brenda after #2898 merged.
bug-focus-beh

## Browser

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [ ] Windows
- [ ] Ubuntu

### PR DESCRIPTION
**Pull request by yeze322**
## Description
closes #2875 

## Task Item

image

image

### CODE REVIEW

File: Composer/packages/lib/ui-shared/src/styled/sharedCSS.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ export const TruncatedCSS = css`
 export const MultilineCSS = css`
   white-space: initial;
   overflow-wrap: break-word;
-  word-break: break-all;
+  word-break: break-word;
```
[the remark this answers is not in the document]
Comment (a-b-r-o-w-n) [reply Composer/packages/lib/ui-shared/src/styled/sharedCSS.tsx:22]: @beyackle would you mind pushing a fix for that and verifying the behavior is the same? I'd like to get this merged today.

[the remark this answers is not in the document]
Comment (beyackle) [reply Composer/packages/lib/ui-shared/src/styled/sharedCSS.tsx:22]: image

Works just fine (text from an MDN page I had open at the time). I'll push this up here.

[the remark this answers is not in the document]
Comment (a-b-r-o-w-n) [reply Composer/packages/lib/ui-shared/src/styled/sharedCSS.tsx]: Thanks @beyackle

[the remark this answers is not in the document]
Comment (yeze322) [reply Composer/packages/lib/ui-shared/src/styled/sharedCSS.tsx]: Thanks @beyackle !

**Pull request merged**